packages/sentry/index.tsx

```diff
@@ -27,3 +29,8 @@ export const sentryFetch: typeof fetch = async (resource, options) => {
          throw error;
       });
 };
+
+export const setSentryPageContext = (context: GetServerSidePropsContext) => {
+   Sentry.configureScope((scope) => scope.clear());
```

Comment:
This is a little worrisome since the mutation-based `configureScope` will blow other data away:

```js
Sentry.withScope(scope => {
	scope.SetTag('ShopifyLoaded', 'true');
	loadPage();
	
	// ShopifyLoaded was just unset
});

function loadPage() {
	setSentryPageContext();
}
```

Reply:
https://github.com/iFixit/react-commerce/pull/600/commits/16b6857716d04c697dc36b085f54b3af0ecc9a2f